Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rust:latest
+
+WORKDIR /app
+
+COPY . .
+
+# Cache download
+RUN cargo fetch 
+
+RUN cargo build --release --target-dir /usr/local/cargo
+
+RUN cargo install diesel_cli
```

Comment:
Does diesel cli need to be installed via cargo / does it need rust installed.
To reduce the size of the final image it would be ideal if only the building container had rust and the runner was a tiny one with only the required binaries.

Reply:
Database setup (uses diesel which apparently uses rust) will run when the image is ran. So at the moment I do not know how to remove rust nor diesel from the final image.

Reply:
Alright, that's fine I guess.